### PR TITLE
fix(test): don't error on missing op details

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes https://github.com/denoland/deno/issues/13938
Closes https://github.com/denoland/deno/issues/13729

### CODE REVIEW

File: runtime/js/40_testing.js [hunk 1]
```diff
@@ -173,13 +173,21 @@
           preOp.opsCompletedAsync;
 
         if (dispatchedDiff > completedDiff) {
-          const [name, hint] = OP_DETAILS[key];
+          const maybeDetails = OP_DETAILS[key];
+          let name = key;
+          let hint = null;
+          if (maybeDetails) {
+            [name, hint] = maybeDetails;
+          }
```
Comment (littledivy) [inline runtime/js/40_testing.js:181]:
```suggestion
          const maybeDetails = OP_DETAILS[key] || [key, null];
```
Maybe?

File: runtime/js/40_testing.js [hunk 2]
```diff
@@ -173,13 +173,16 @@
...
         if (dispatchedDiff > completedDiff) {
-          const [name, hint] = OP_DETAILS[key];
+          const [name, hint] = OP_DETAILS[key] || [key, null];
           const count = dispatchedDiff - completedDiff;
           let message = `${count} async operation${
             count === 1 ? "" : "s"
           } to ${name} ${
             count === 1 ? "was" : "were"
-          } started in this test, but never completed. This is often caused by not ${hint}.`;
+          } started in this test, but never completed.`;
```
Comment (aapoalas) [inline runtime/js/40_testing.js:182]: Personally I would go with an infernal nested template string:
```ts
`...but never completed.${hint ? ` This is often caused by not ${hint}.` : ""}
```
Buuut that's just personal preference and I admit that nested template strings are... uhm, not pretty to put it mildly.

Comment (aapoalas) [inline runtime/js/40_testing.js:182]: Personally I would go with an infernal nested template string:
```ts
`...but never completed.${hint ? ` This is often caused by not ${hint}.` : ""}
```
Buuut that's just personal preference and I admit that nested template strings are... uhm, not pretty to put it mildly.

**Comment by aapoalas** [comment]
LGTM

**Review by dsherret — APPROVED** [review]
LGTM

**Pull request merged**